lib/resources/aws/aws_iam_policy.rb

```diff
@@ -50,8 +69,133 @@ def attached_to_role?(role_name)
     attached_roles.include?(role_name)
   end
 
+  def policy
+    return nil unless exists?
+    return @policy if defined?(@policy)
+
+    catch_aws_errors do
+      backend = BackendFactory.create(inspec_runner)
+      gpv_response = backend.get_policy_version(policy_arn: arn, version_id: default_version_id)
+      @policy = JSON.parse(URI.decode_www_form_component(gpv_response.policy_version.document))
+    end
+    @policy
+  end
+
+  def statement_count
+    return nil unless exists?
+    policy['Statement'].count
```

Comment:
Will statement always be populated?

Reply:
Yes - it's invalid otherwise, and AWS won't accept an invalid policy JSON document.